test/models/vrp_consistency_test.rb

```diff
@@ -305,7 +305,7 @@ def test_duplicated_ids_are_not_allowed
 
     def test_dates_cannot_be_mixed_with_indices
       vrp = VRP.scheduling # contains schedule[:range_indices]
-      vrp[:vehicles].first[:unavailable_work_date] = Date.new(2021, 2, 11)
+      vrp[:vehicles].first[:unavailable_work_date] = [Date.new(2021, 2, 11)]
```

Comment:
Comment out of the scope of this PR:

This parameter is an array, it should have been called `unavailable_work_dates` :facepalm: 

Is this the parameter we recently replaced with `unavailable_x_ranges`? Or this one does something different?

IF it is the one that is "replaced" and we want people to use the  `unavailable_date/index_ranges` logic then we can make `unavailable_work_day_indices` and `unavailable_work_date` hidden in the document. IF we want to keep it, we should hide/deprecate this one and add a `unavailable_work_dates` parameter to the public API.

@fonsecadeline I think you are more informed about the use-cases so I think you should decide... can you create a ticket accordingly?

Reply:
https://gitlab.com/mapotempo/optimizer-api/-/issues/704